index.html

```diff
@@ -349,7 +349,9 @@
   A Thing Description instance can be hosted by the <a>Thing</a> itself or hosted
   externally when a <a>Thing</a> has resource restrictions (e.g., limited memory space)
   or when a Web of Things-compatible legacy device is retrofitted 
-  with a Thing Description.
+  with a Thing Description. This specification is a continuation and further development 
+  of the Thing Description 1.0 [[WOT-THING-DESCRIPTION]]. 
+  It has the claim that the Thing Description 1.1 is backward compatible to Thing Description 1.0.  
```

Comment:
Maybe we can simplify it by saying:

Web of Things Thing Description 1.1 is backwards compatible with Thing Description 1.0 [[WOT-THING-DESCRIPTION]].

Reply:
The text is replaced by: "This specification describes a superset of the features defined in Thing Description 1.0. Unless otherwise specified, documents created with version 1.0 of this specification remain compatible with Thing Description 1.1."